openwrt-openwisp-monitoring/Makefile

```diff
@@ -19,7 +19,7 @@ define Package/openwisp-monitoring/default
 	CATEGORY:=Administration
 	SECTION:=admin
 	SUBMENU:=openwisp
-	DEPENDS:=+netjson-monitoring $(3)
+	DEPENDS:=+netjson-monitoring +gzip $(3)
```

Comment:
wait, we don't need the full gzip, we should be able to use the gzip utility provided by busy box, which is shipped by default in OpenWrt:

```
gzip --help
BusyBox v1.33.1 (2021-07-31 21:08:50 UTC) multi-call binary.

Usage: gzip [-cfkdt] [FILE]...

Compress FILEs (or stdin)

	-d	Decompress
	-t	Test file integrity
	-c	Write to stdout
	-f	Force
	-k	Keep input files
``` 

We are space constrained on these devices, we have to be extra careful before adding a new dependency.

Reply:
If `gzip` is already available, this shouldn't take any extra space. However, I removed it as I am not sure if the gzip package is heavier than the one provided by busy box